src/App.js

```diff
@@ -1,59 +1,87 @@
 import './App.css';
 import Header from './Components/Header';
 import Footer from './Components/Footer';
-import { useEffect, useState } from "react";
-import { Routes, Route } from "react-router-dom";
+import { useEffect, useState } from 'react';
+import { Routes, Route } from 'react-router-dom';
 import MainPage from './pages/MainPage';
 import DetailedPage from './pages/DetailedPage';
 import RandomPage from './pages/RandomPage';
 import FavoritesPage from './pages/FavoritesPage';
 import CreativePage from './pages/CreativePage';
-  
-
 
 function App() {
-
   const [characters, setCharacters] = useState([]);
-  const [favoriteCharacters, setFavoriteCharacters] = useState([])
+  const [favoriteCharacters, setFavoriteCharacters] = useState([]);
 
-  const url = "https://rickandmortyapi.com/api/character/";
+  const url = 'https://rickandmortyapi.com/api/character/';
 
   const loadCharacters = () => {
     fetch(url)
-      .then((response) => response.json())
-      .then((data) => setCharacters(data.results));
+      .then(response => response.json())
+      .then(data => setCharacters(data.results));
   };
 
   useEffect(() => {
     loadCharacters();
   }, []);
 
-  const handleBookmarking = (id) => {
+  const loadfavoriteCharacters = () => {
+    const localStorageReturn = JSON.parse(localStorage.getItem('favChar'));
+    localStorageReturn === null
```

Comment:
It is confusing to use the ternary operator here, since there is no variable assignment / return value from the setFavoriteCharacters function. It's better to use if else here.